polyfill/spatial-navigation-polyfill.js

```diff
@@ -1200,6 +1237,44 @@
     return (A + B - C - D);
   }
 
+  /**
+   * Get the euclidean distance between the search origin and a candidate element considering the direction.
+   * @function getAbsoluteDistance
```

Comment:
This needs to be change to "getEuclideanDistance"

Reply:
There was mistake in the annotation. I'll fix.